task/06-conditions-n-loops-tasks.js

```diff
@@ -356,9 +423,14 @@ function timespanToHumanString(startDate, endDate) {
  *    365, 10 => '365'
  */
 function toNaryString(num, n) {
-    throw new Error('Not implemented');
+    var result = '';
+    while (num !== 0) {
+        result = (num % n) + result;
+        num = Math.floor(num / n);
+    }
+    return result;
 }
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString

Reply:
Ok. It is a really simpler,but one usefull thing - I have bethought how numbers converted)